Add CREDITS.md (#69)
duckinator/emanate: CREDITS.md

> # Emanate Maintainers

Reviewer: I think we can shorten it:
```suggestion
# Maintainers
```

- [Ellen Marie Dash (@duckinator)](https://github.com/duckinator)
- [nicoo (@nbraud)](https://github.com/nbraud)

> # Emanate Contributors

Reviewer: Same
```suggestion
# Contributors
```

These are folks who aren't maintainers, but have contributed in some way. :)

- [Samuel Lewis (@ritlew)](https://github.com/ritlew)

> 

Reviewer: ```suggestion
- [@kayila](https://github.com/kayila)

```

She caused some interesting/useful discussion in #54 (which I should get off my butt and act on).

# Software and Services We Rely On ♥

All of the following help make Emanate as awesome as it is:

Software and libraries:

- [Python](https://python.org)

> - [flake8](https://gitlab.com/pycqa/flake8)

Reviewer: ```suggestion
- [flake8](https://gitlab.com/pycqa/flake8) and its [bugbear](https://github.com/PyCQA/flake8-bugbear), and [commas](https://github.com/PyCQA/flake8-commas/) plugins;
- [pydocstyle](https://github.com/PyCQA/pydocstyle/), though the [docstrings](https://gitlab.com/pycqa/flake8-docstrings) plugin for flake8;
```

Reviewer: Perhaps don't use inline links here, it's getting kinda unwieldy?

- [pip](https://pip.pypa.io/)

> - [pylint](https://www.pylint.org/)

Reviewer: ```suggestion
- [pylint](https://www.pylint.org/) and the [pytest-pylint](https://github.com/carsongee/pytest-pylint) integration;
```

- [pytest](https://pytest.org/)

> - [setuptools](https://github.com/pypa/setuptools)

Reviewer: ```suggestion
- [setuptools](https://github.com/pypa/setuptools) and [wheel](https://github.com/pypa/wheel);
```

# Maintainers

- [Ellen Marie Dash (@duckinator)](https://github.com/duckinator)
- [nicoo (@nbraud)](https://github.com/nbraud)

# Contributors

These are folks who aren't maintainers, but have contributed in some way. :)

- [Samuel Lewis (@ritlew)](https://github.com/ritlew)
- [@kayila](https://github.com/kayila)

# Software and Services We Rely On ♥

All of the following help make Emanate as awesome as it is:

Software and libraries:

- [Python](https://python.org)
- [flake8](https://gitlab.com/pycqa/flake8)
  - flake8 [bugbear](https://github.com/PyCQA/flake8-bugbear) plugin
  - flake8 [commas](https://github.com/PyCQA/flake8-commas/) plugin

> - [pydocstyle](https://github.com/PyCQA/pydocstyle/) through the [docstrings](https://gitlab.com/pycqa/flake8-docstrings) plugin for flake8

Reviewer: ```suggestion
  - [docstrings](https://gitlab.com/pycqa/flake8-docstrings) plugin to integrate with [pydocstyle](https://github.com/PyCQA/pydocstyle/)
```
